Src/Base/AMReX_FabArrayBase.cpp

```diff
@@ -671,7 +675,8 @@ FabArrayBase::FB::define_fb(const FabArrayBase& fa)
     {
 	const int ksnd = imap[i];
 	const Box& vbx = ba[ksnd];
-	
+	const Box& vbx_ng  = amrex::grow(ba[ksnd],1);
```

Comment:
It is slightly more efficient to do `const Box& vbx_ng = amrex::grow(vbx,1);` to avoid the cost of `BoxArray::operator[]`.

Reply:
Done